java/server/src/org/openqa/selenium/grid/data/DistributorStatus.java

```diff
@@ -203,7 +210,7 @@ private static NodeSummary fromJson(JsonInput input) {
 
       input.endObject();
 
-      return new NodeSummary(nodeId, uri, up, maxSessionCount, stereotypes, used);
+      return new NodeSummary(nodeId, uri, up, maxSessionCount, stereotypes, used, null);
```

Comment:
Pass in an empty set rather than `null` and you'll avoid null pointer exceptions later